mock-continuum-login/src/use-cases/submit.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+import { sign } from "jsonwebtoken";
+
+export const Submit = (req: Request, resp: Response) => {
+    const token = sign({
+        iss: "journal--prod",
+        id: "ewwboc7m",
+        "new-session": true
+    },
+    'some_secret_from_journal', {
+        expiresIn: '1d',
```

Comment:
I think the expiry from journal is longer than this? @petereast ?

Reply:
https://github.com/elifesciences/elife-xpub/blob/c383f75960c03febf387123d80409df3c63252bc/packages/component-elife-app/server/resolvers.js#L15

we have it at one day currently